scripts/seed_db.py

```diff
@@ -0,0 +1,160 @@
+import csv
+import re
+import requests
+import json
+import time
+
+# Assumption: Given A1 -> N15
+#  A1 ............ N15            1  16 ........ 196
+#  A2              N2             2  17          197
+#  .               .              .  .           .
+#  .               .       --->   .  .           .
+#  .               .              .  .           .
+#  A15 ..........  N15            15 30 ........ 210
+
+
+# gdi_devpost -> csv is dumb lmao
+# moving -> list of projects that can move
+# notMoving -> list of projects that can't move
+# spots -> dict that goes table number:hack name
+# assignments -> list of spots that are taken
+gdi_devpost = "Does Your Hack Need To Stay At Your Current Table? " \
+              "(I.E. Hardware, Vr/Ar Hacks). If So, What Table" \
+              " Number Are You At?"
+moving, notMoving = {}, {}
+assignments = ["None | "] * 391
```

Comment:
What's the 391 for/can we genericize it to work with any sized csv?

Reply:
392 is just a 14x28 rectangle